docs/jira-setup-guide.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Jira Environment Variable Setup Guide
+layout: basic-page-with-footer-links
+pageDescription: On this page, you will find a step-by-step guide on how to setup the JF agent to work with Jira.
+nextPage: Step 3&#58; Ensure proper network configuration
+nextPageLink: network-config-setup-guide.html
+previousPage: Step 2&#58; Create environment variable file
+previousPageLink: env-var-setup-guide.html
+---
+
+
+## Details
+
+This is where you'll add and populate your environmental variables file for Jira.  If you want your Git instance to work with the agent, see the setup guide corresponding to the type of Git instance you have - it will include Jira instructions as well.
+
+
+## Instructions
+
+1. Add the following section to your environment variable file. This is the same file mentioned [in step 2](env-var-setup-guide.html). Adding the following variables allows the agent to access your Jira data:
+    <p class="code-block"><code>
+        JIRA_USERNAME=...<br/>
+        JIRA_PASSWORD=...
+    </code></p>
```

Comment:
one more quick question: does this come out properly in the HTML or should it be converted into markdown style code blocks? e.g.

    ```
    JIRA_USERNAME=...
    JIRA_PASSWORD=...
    ```

Reply:
Here is what it looks like without these tags

<img width="1310" alt="Screen Shot 2021-07-29 at 11 56 43 AM" src="https://user-images.githubusercontent.com/29586620/127525138-421cc155-3ec0-40f8-979d-53af9cdbbc9d.png">

Reply:
The longer code blocks are probably a better example
<img width="1375" alt="Screen Shot 2021-07-29 at 12 00 36 PM" src="https://user-images.githubusercontent.com/29586620/127525654-804b17c7-0eec-4e84-b208-769f311894a0.png">